f3dex2.s

```diff
@@ -52,6 +52,8 @@ overlay_imem equ 0x0006
     .dh addr & 0xFFFF
 .endmacro
 
+DMEM equ $zero // Makes it easier to tell, at a glance, if a load/store comes from dmem directly.
```

Comment:
I don't think this is too helpful since all loads/stores come from DMEM, the hardware doesn't allow any other target for loads/stores. (the address computed by loads/stores is only 12-bit so can only address 0 to 0xFFF inclusive)
Further, a lot of `($zero)` is not written explicitly, since the assembler will use $zero as the base by default if not written. The only reason this isn't done for vector load/stores is because of an armips bug(?) that doesn't allow the base register to be unspecified.